modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/MultiActorPlacementDriverTest.java

```diff
@@ -288,31 +284,26 @@ public List<Closeable> startPlacementDriver(
             clusterService.start();
             raftManager.start();
             metaStorageManager.start();
-            placementDriverManager.start();
-
-            msFutures[i] = metaStorageManager.deployWatches();
-
-            res.add(() -> {
-                        try {
-                            placementDriverManager.beforeNodeStop();
-                            metaStorageManager.beforeNodeStop();
-                            raftManager.beforeNodeStop();
-                            clusterService.beforeNodeStop();
-                            vaultManager.beforeNodeStop();
-
-                            placementDriverManager.stop();
-                            metaStorageManager.stop();
-                            raftManager.stop();
-                            clusterService.stop();
-                            vaultManager.stop();
-                        } catch (Exception e) {
-                            log.info("Fail to stop services [node={}]", e, nodeName);
-                        }
-                    }
-            );
+
+            res.add(new Node(nodeName, vaultManager, clusterService, raftManager, metaStorageManager, placementDriverManager));
         }
 
-        assertThat("Nodes were not started", CompletableFuture.allOf(msFutures), willCompleteSuccessfully());
+        assertThat(
```

Comment:
As mentioned above, I believe that it'l be better to hide all that assertions and component start awaiting logic inside new Node class.

Reply:
Did it